packages/utils/src/node/sentry-logger.ts

```diff
@@ -19,7 +19,7 @@ export const createWistonLoggerErrorFn: (loggerError: LeveledLogMethod) => Wisto
   { error, traceId, headers },
 ) =>
   new Promise(resolve => {
-    loggerError(caller, { traceId, error: error?.response?.data, headers: error?.response?.headers })
+    loggerError(caller, { traceId, error: JSON.stringify(error) })
```

Comment:
Can we use 
```
console.log(JSON.stringify(error, null, 2));
`` for readability?
https://stackoverflow.com/questions/24834812/space-in-between-json-stringify-output

Reply:
Thanks ![image](https://user-images.githubusercontent.com/36730355/92433892-8aa45480-f1c8-11ea-9c61-a96696081cb5.png)